src/Compilers/VisualBasic/Test/Emit/Attributes/InternalsVisibleToAndStrongNameTests.vb

```diff
@@ -1639,9 +1639,11 @@ End Class
         Assert.True(compilation.Options.PublicSign)
         Assert.Null(compilation.Options.DelaySign)
         compilation.VerifyDiagnostics()
```

Comment:
It looks like VerifyEmitDiagnostics call is redundant as well because the next thing we do is call Emit, we can simply check diagnostics on EmitResult.